boards/shields/waveshare_epaper/waveshare_epaper_gdew042t2-p.overlay

```diff
@@ -8,87 +8,93 @@
 
 / {
 	chosen {
-		zephyr,display = &uc8176_waveshare_epaper_gdew042t2-p;
+		zephyr,display = &uc8176_waveshare_epaper_gdew042t2_p;
```

Comment:
Unintentional?

Reply:
Without this change I was seeing a build error like so:
```
devicetree error: /home/danieldegrasse/zephyr/zephyrproject/zephyr/boards/shields/waveshare_epaper/waveshare_epaper_gdew042t2-p.overlay:11 (column 54): parse error: expected ';' or ','
```
with the shield- I don't think we can have dashes in the nodelabel name, so I think this shield might have been broken upstream

Reply:
`-` is not a valid character for a label according to the specification (only 0-9, a-z, A-Z, _).